protocol/xmrtaker/swap_state_test.go

```diff
@@ -255,8 +259,12 @@ func TestSwapState_NotifyXMRLock(t *testing.T) {
 	xmrmakerKeysAndProof, err := generateKeys()
 	require.NoError(t, err)
 
-	s.setXMRMakerKeys(xmrmakerKeysAndProof.PublicKeyPair.SpendKey(), xmrmakerKeysAndProof.PrivateKeyPair.ViewKey(),
-		xmrmakerKeysAndProof.Secp256k1PublicKey)
+	err = s.setXMRMakerKeys(
+		xmrmakerKeysAndProof.PublicKeyPair.SpendKey(),
+		xmrmakerKeysAndProof.PrivateKeyPair.ViewKey(),
+		xmrmakerKeysAndProof.Secp256k1PublicKey,
+	)
+	require.NoError(t, err)
```

Comment:
These changes just add a check on the returned error value. My IDE was complaining.